package.json

```diff
@@ -17,6 +17,11 @@
     "test": "BABEL_ENV=test mocha --require test/config/unit-test-setup.js --require test/config/enzyme-setup.jsx --compilers js:babel-core/register ./test/**/*.spec.jsx",
     "webpack": "webpack"
   },
+  "husky": {
+    "hooks": {
+      "pre-push": "npm run build && git diff --quiet || echo '** NOT PUSHED: Uncommitted changes! **'"
```

Comment:
Hm, can we not have the pre-push script just run the build itself as opposed to just warning someone that it hasn't yet been run? It would just save a step if it runs the build automatically. What do you think?

Reply:
Oops! Totally misread this. This looks good to me!